src/test/java/com/exadel/discountwebapp/role/RoleServiceIntegrationTest.java

```diff
@@ -15,7 +15,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-@Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts = "classpath:sql/cleanup.sql")
+@Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts = "classpath:sql/clean-up.sql")
 @SpringBootTest
```

Comment:
if you wanna execute "category-init.sql" file maybe it need be added as @Sql(scripts = "classpath:schema.sql")?